evandana/keep-ahope
# Contact info card

# Description

Add functionality for displaying contact info

# To test
- navigate to /contacts/ > contact info tab
- see two new cards in addition to existing view
- Profile card should have data from firebase
